citesphere-model/src/main/java/edu/asu/diging/citesphere/data/bib/ICitationDao.java

```diff
@@ -15,6 +15,17 @@ public interface ICitationDao {
 
     List<? extends ICitation> findCitationsInCollection(String groupId, String collectionId, long start, int pageSize, List<String> conceptIds);
 
+    /**
+     * This method returns all citations of an user based on the
+     * query param uri.
```

Comment:
that doesn't say a lot, I can see that from the parameters. javadoc should tell me what exactly the method returns. Also, it should be "a user" not "an user".